app/src/main/java/com/example/myapplication/Constants.java

```diff
@@ -8,10 +8,10 @@
     static final String ORDERS = "OpenOrders";
     static final String MANOT_SUBCOLLECTION = "Manot";
     static final String MONEY_MADE = "הכסף שנצבר: %s מתוך 70 שקלים";
-    static final String LOCKED_TEXT = "נעול";
+    static final String LOCKED_TEXT = "הזמן!";
     static final String JOIN_TEXT = "הצטרף!";
     static final String READY_TEXT = "מוכן לשילוח";
-    static final String ORDER_OUT = "ההזמנה יצאה";
+    static final String ORDER_OUT = "שעת ההזמנה הגיעה. מישהו צריך לקחת יוזמה";
```

Comment:
Same here

Reply:
You're right, I'll take care of it after merging.